sites/docs/src/content/docs/tutorials/switching_barnch_to_main/switching_master_to_main.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "Transitioning from `master` to `main` branches in pipelines"
+subtitle: Guide on how to switch a pipeline from the default branch `master` to `main`
+weight: 20
+---
+
+This tutorial will guide you through changing the pipeline default branch form `master` to `main`.
+
+## Prequisites
+
+- Admin rights on the GitHub repository where the change taking place
+- nf-core tools
+  - It is recommended that the version match the version of the template in your pipeline
+
+> [!WARNING]
+> Make sure you are on using same version of nf-core/tools as the template version of your branch.
+> This will avoid adding other changes not needed for the branch name switch.
```

Comment:
```suggestion
:::warning
Make sure you are on using the same version of nf-core/tools as the template version of your branch (you can see the version in `.nf-core.yml`).
This will avoid adding other changes not needed for the branch name switch.
:::
```